aspnetcore/blazor/tutorials/movie-database-app/part-3.md

```diff
@@ -0,0 +1,613 @@
+
+### `Create` component
+
+Open the `Create` component definition file (`Components/Pages/Movies/Create.razor`).
+
+The component uses a built-in component called an <xref:Microsoft.AspNetCore.Components.Forms.EditForm>, which renders a form for user input and includes validation features.
+
+CSS classes aren't present in the following example to simplify the display:
+
+```razor
+<EditForm method="post" Model="Movie" OnValidSubmit="AddMovie" FormName="create" Enhance>
+    <DataAnnotationsValidator />
+    <ValidationSummary />
+    <div>
+        <label for="title">Title:</label> 
+        <InputText id="title" @bind-Value="Movie.Title" /> 
+        <ValidationMessage For="() => Movie.Title" /> 
+    </div>
+    <div>
+        <label for="releasedate">ReleaseDate:</label> 
+        <InputDate id="releasedate" @bind-Value="Movie.ReleaseDate" /> 
+        <ValidationMessage For="() => Movie.ReleaseDate" /> 
+    </div>
+    <div>
+        <label for="genre">Genre:</label> 
+        <InputText id="genre" @bind-Value="Movie.Genre" /> 
+        <ValidationMessage For="() => Movie.Genre" /> 
+    </div>
+    <div>
+        <label for="price">Price:</label> 
+        <InputNumber id="price" @bind-Value="Movie.Price" /> 
+        <ValidationMessage For="() => Movie.Price" /> 
+    </div>
+    <button type="submit">Create</button>
+</EditForm>
+```
+
+The <xref:Microsoft.AspNetCore.Components.Forms.EditForm.Model%2A> parameter is assigned the model, in this case `Movie`. <xref:Microsoft.AspNetCore.Components.Forms.EditForm.OnValidSubmit%2A> specifies a method to invoke (`AddMovie`) when the form is submitted. By convention, every form should assign a <xref:Microsoft.AspNetCore.Components.Forms.EditForm.FormName%2A> to prevent form collisions when multiple forms are present on a page. The <xref:Microsoft.AspNetCore.Components.Forms.EditForm.Enhance%2A> flag activates a Blazor feature for server-side rendering (SSR) that submits the form without performing a full-page reload.
+
+For validation:
+
+* The <xref:Microsoft.AspNetCore.Components.Forms.DataAnnotationsValidator> adds data annotations validation support, which is covered later in this tutorial series.
+* The <xref:Microsoft.AspNetCore.Components.Forms.ValidationSummary> component displays a list of validation messages.
+* The <xref:Microsoft.AspNetCore.Components.Forms.ValidationMessage%601> components hold validation messages for the form's fields.
+
+Blazor includes several form element components to assist you with creating forms. The <xref:Microsoft.AspNetCore.Components.Forms.EditForm> form uses <xref:Microsoft.AspNetCore.Components.Forms.InputText>, <xref:Microsoft.AspNetCore.Components.Forms.InputDate%601>, and <xref:Microsoft.AspNetCore.Components.Forms.InputNumber%601> components. Each of these is bound to a model property with `@bind-Value` Razor syntax, where `Value` is a property in each input component.
```

Comment:
This was a wording 😈. I should've placed the word "preceding" before `EditForm` because this was specifically referring to ***this*** component generated by the scaffolder, not to any given `EditForm`.